ehrenamt-justiz-frontend/src/api/EhrenamtJustizService.ts

```
@@ -273,6 +273,13 @@ class EhrenamtJustizServiceClass {
             globalSettingsStore.getKonfiguration?.ehrenamtjustizart +
             " tätig?",
         },
+        {
+          key: "warbereitstaetigalsvorvorperiode",
+          displayLabel:
+            "War bereits als " +
+            globalSettingsStore.getKonfiguration?.ehrenamtjustizart +
+            " in Vorvorperiode tätig?",
+        },
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)